src/views/Project/index.jsx

```diff
@@ -32,15 +51,57 @@ const tagReposMapping = repositories =>
   }, {});
 
 @hot(module)
-@graphql(issuesQuery, {
-  skip: props => !projects[props.match.params.project].repositories,
-  options: () => ({
-    fetchPolicy: 'network-only',
-    variables: {
-      searchQuery: '',
-    },
+@compose(
+  graphql(issuesQuery, {
+    skip: ({
+      match: {
+        params: { project },
+      },
+    }) => !projects[project].repositories,
+    name: 'github',
+    options: () => ({
+      fetchPolicy: 'network-only',
+      variables: {
+        searchQuery: '',
+      },
+      context: {
+        client: 'github',
+      },
+    }),
   }),
-})
+  graphql(bugsQuery, {
+    skip: ({
+      match: {
+        params: { project },
+      },
+    }) => !projects[project].products,
+    name: 'bugzilla',
+    options: ({
+      match: {
+        params: { project },
+      },
+    }) => ({
+      fetchPolicy: 'network-only',
+      variables: {
+        search: {
+          ...bugzillaSearchOptions,
+          ...(productsWithNoComponents(projects[project].products).length
+            ? { products: productsWithNoComponents(projects[project].products) }
+            : {
+                products: Object.keys(projects[project].products[0])[0],
+                components: Object.values(projects[project].products[0])[0],
```

Comment:
Could you add a comment explaining **why** we are only grabbing the first entry of the array? This will help people hacking on this in the future to quickly understand.